pkgs/by-name/co/cosmic-term/package.nix

```
@@ -1,44 +1,44 @@
-{
-  lib,
-  stdenv,
-  fetchFromGitHub,
-  rust,
-  rustPlatform,
-  cmake,
-  makeBinaryWrapper,
-  cosmic-icons,
-  just,
-  pkg-config,
-  libxkbcommon,
-  libinput,
-  fontconfig,
-  freetype,
-  wayland,
-  xorg
+{ lib
+, cosmic-icons
+, fetchFromGitHub
+, fontconfig
+, freetype
+, just
+, libglvnd
+, libinput
+, libxkbcommon
+, makeBinaryWrapper
+, mesa
+, pkg-config
+, rustPlatform
+, stdenv
+, vulkan-loader
+, wayland
+, xorg
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cosmic-term";
-  version = "unstable-2023-12-26";
-
+  version = "0-unstable-2024-01-19";
```

Unstable versions should not start with any digits, so that it doesn't have weird version sorting.
```suggestion
  version = "unstable-2024-01-19";
```